src/core/model.js

```diff
@@ -405,6 +410,17 @@ in a future version. Use shader modules instead.`);
     return this;
   }
 
+  // HELPER METHODS
+
+  _checkForDeprecatedUniforms(uniforms) {
```

Comment:
@ibgreen 
I am wondering do we have to do this, what if an app uses these uniforms for any other purpose?

Reply:
I think it is OK. It should just generate a single warning. Helps us catch any stragglers and we can remove in next release.